source/source.go

```diff
@@ -87,7 +87,7 @@ type Config interface {
 // sources contain all registered sources
 var sources = make(map[string]Source)
 
-// RegisterSource registers a source
+// Register expects a source and registers a source
```

Comment:
nit: I think *expects a source* is pretty self-evident

```suggestion
// Register registers a source.
```

Reply:
changed